src/elements/content-sidebar/activity-feed/comment/RepliesToggle.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+import PlainButton from '../../../../components/plain-button';
+
+import './RepliesToggle.scss';
+
+type Props = {
+    onHideReplies: (index: number) => void,
+    onShowReplies: () => void,
+    repliesShownCount: number,
+    repliesTotalCount: number,
+};
+
+const RepliesToggle = ({ onShowReplies, onHideReplies, repliesShownCount, repliesTotalCount }: Props) => {
+    if (repliesTotalCount <= 1) {
+        return null;
+    }
+
+    const hasMoreRepliesToShow = repliesTotalCount > repliesShownCount;
+    const toggleMessage = hasMoreRepliesToShow ? messages.showReplies : messages.hideReplies;
+    const repliesToLoadCount = Math.max(repliesTotalCount - repliesShownCount, 0);
+
+    const handleToggle = () => {
+        if (hasMoreRepliesToShow) {
+            onShowReplies();
+        } else if (repliesShownCount) {
+            onHideReplies(repliesShownCount - 1);
```

Comment:
Does this match how onHideReplies is currently being used? How do you plan on using repliesShownCount - 1 to actually hide the replies?

Reply:
Yes it's going to be working the same way it's currently being used. The toggle passes back the last index shown, and Replies component will pass back the last reply shown.

Reply:
This looks good for now. It matches the current functionality. Let's revisit how we are showing/hiding the replies as we integrate it with the other components and see if there's a way of simplifying that process.